reconcile/change_owners/change_owners.py

```diff
@@ -120,7 +121,71 @@ def fetch_bundle_changes(comparison_sha: str) -> list[BundleFileChange]:
     explicitely passed comparision bundle - usually the state of the master branch).
     """
     changes = gql.get_diff(comparison_sha)
-    return _parse_bundle_changes(changes)
+    bundle_changes = _parse_bundle_changes(changes)
+    try:
+        return _aggregate_file_moves(bundle_changes)
+    except Exception as e:
```

Comment:
What exception are we expecting here?

Reply:
this tries to recover from a situation where the metadata retrieved from qontract-server would not be valid, e.g. sha checksum missing (`InvalidBundleFileMetadataError`)

i reconsidered though and do not catch the exception anymore. if qontract-server would really not return valid metadata (or any other error situation) it is best to fail fast.